plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/thrift/InMemoryThriftMetastore.java

```diff
@@ -326,6 +327,24 @@ public synchronized List<String> getAllViews(String databaseName)
         return tables.build();
     }
 
+    @Override
+    public Map<String, List<String>> getAllTables()
+    {
+        return this.relations.keySet().stream()
+                .map(fullName -> fullName.getSchemaName())
+                .distinct()
+                .collect(toImmutableMap(schema -> schema, schema -> getAllTables(schema)));
```

Comment:
nit: Can also use `identity` function here.

Reply:
We can move to a functional APIs here.

Reply:
Can we instead of using `Relation` can we use `Databases` here ?

Reply:
I used `relations` to be logically identical to `getAllTables(dbName)` method.